### PR TITLE
Add a tracking of global loss in each epoch

### PR DESCRIPTION
**Pull request by ducovrossem**
Something to look at while training the model :)

```
global_loss += 0.5 * entry_weight * (prediction - c_log(count)) **2
```

### CODE REVIEW

**Comment by IronFarm** [comment]
I know this pull request has gotten stale but is there any interest in getting it merged? I've managed to merge it into master locally and would be willing to fork and open a new pull request where we can discuss it. Three years have passed but it still looks like a valuable addition!

**Comment by gokceneraslan** [comment]
It'd be really great to add this feature!
